webcompat/templates/home-page/issue-wizard-form.html

```diff
@@ -196,18 +196,18 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
     <div class="row half centered other-browser form-element js-Form-groupform-element js-Form-group with-validation-icons low">
       <div class="column full-width">
         <div class="label">
-          {{ form.other_browser.label(class_='form-label') }}
+          Browser tested
```

Comment:
Is there any reason to move the label out of form.py here?

Reply:
@miketaylr form.other_browser.label is also used in the A version of the form, so by changing it we change it for both versions. As far as I know, with WTForms we cannot have multiple labels for the same field, therefore the static label.

Reply:
Makes sense, thanks. I guess the other option would be to do something silly like `other_browser_b`. But I don't think we consume the label on the backend, so it doesn't matter much.